Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1096,6 +1125,26 @@ void ACesium3DTileset::Serialize(FArchive& Ar) {
   }
 }
 
+#if WITH_EDITOR
+void ACesium3DTileset::PostEditChangeProperty(
+    FPropertyChangedEvent& PropertyChangedEvent) {
+  const FName PropName = (PropertyChangedEvent.Property)
+                             ? PropertyChangedEvent.Property->GetFName()
+                             : NAME_None;
+  if (PropName == GET_MEMBER_NAME_CHECKED(ACesium3DTileset, TilesetSource) ||
```

Comment:
Is this necessary? Because when the property is set from Blueprints, it'll go through the setter anyway, right? Which marks the tileset dirty. Or have I missed something?

Reply:
The PostEditChangeProperty method override ensures that editor-time (GUI) changes to these properties will mark the dirty flag. I believe the setters won't get called in that scenario, and the OnConstruction method will call LoadTileset() but the dirty flag will still have been set to false from the previous load.

Reply:
Oh ok, for some reason I expected the editor would go through the blueprint getters/setters too.